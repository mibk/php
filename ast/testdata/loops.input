<?php function a(){

for    (
  $i = 0
; $i < 10
; $i++ ){
echo "$i\n"
;      }for(;;)echo 'âˆž';

for ($x;;)$x++;
for (;$y;)$y++;
for (;;next())prev();
foreach ( $this-> elems as $el )$el();
foreach ( self::$x as $y) $y?;
}