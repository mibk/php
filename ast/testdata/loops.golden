<?php

function a() {
	for ($i = 0; $i < 10; $i++) {
		echo "$i\n";
	}
	for (;;) echo '∞';
	for ($x;;) $x++;
	for (; $y;) $y++;
	for (;; next()) prev();
}
